### PR TITLE
[ADD] account_invoice_consolidated (#22)

### PR DESCRIPTION
**Pull request by max3903**
#22 

CC: @Swapnesh-SerpentCS, @B-kannan

### CODE REVIEW

**Review by osi-scampbell — APPROVED** [review]

**Comment by Swapnesh-SerpentCS** [comment]
@max3903 
I missed using `commercial_partner_id` here.
(I will add on Monday If you suggest)

(PS: and also Logic to prevent adding same Open invoice to multiple Consolidated Invoice)

File: account_invoice_consolidated/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Balaji Kannan <bkannan@opensourceintegrators.com>
+* Swapnesh Shah <swapnesh.shah@serpentcs.com>
```
Comment (murtuzasaleh) [inline account_invoice_consolidated/readme/CONTRIBUTORS.rst]: * Swapnesh Shah <s.shah.serpentcs@gmail.com>

File: account_invoice_consolidated/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        'views/res_company_view.xml',
+        'views/account_invoice_view.xml',
+        'views/res_partner_view.xml',
+        'views/report_consolidated_invoice.xml',
+    ],
+    'application': True,
+    'development_status': 'Beta',
+    'maintainers': [
+        'max3903',
+        'swapnesh-serpentcs',
```
Comment (murtuzasaleh) [inline account_invoice_consolidated/__manifest__.py:33]: Swapnesh-SerpentCS

File: account_invoice_consolidated/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2019 Open Source Integrators
```
Comment (murtuzasaleh) [inline account_invoice_consolidated/models/__init__.py:1]: Is it needed in __init__.py?

Comment (max3903) [reply account_invoice_consolidated/models/__init__.py:1]: Yes

File: account_invoice_consolidated/__manifest__.py (hunk 2 above)
Comment (max3903) [reply account_invoice_consolidated/__manifest__.py:33]: Github is not case-sensitive...

File: account_invoice_consolidated/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (max3903) [inline account_invoice_consolidated/readme/CONTRIBUTORS.rst]:
```suggestion
* Swapnesh Shah <s.shah.serpentcs@gmail.com>
```

File: account_invoice_consolidated/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
+* Go to Settings > Users & Companies > Companies
+* For each company, go to the "Intercompany Payment Configuration" tab
+* Set
+
+  * the Due From Account
+  * the Due To Account
+  * the Dur From/To Journal
```
Comment (max3903) [inline account_invoice_consolidated/readme/CONFIGURE.rst]:
```suggestion
  * the Due From/Due To Journal
```

**Review by murtuzasaleh — APPROVED** [review]

**Pull request merged**